apps/server/src/modules/learnroom/service/common-cartridge-export.service.spec.ts

```diff
@@ -98,5 +98,14 @@ describe('CommonCartridgeExportService', () => {
 				expect(manifest).toContain(`i${task.id}`);
 			});
 		});
+
+		it('should add copyright information to manifest file', () => {
+			const manifest = archive.getEntry('imsmanifest.xml')?.getData().toString();
+			expect(manifest).toContain(new Date().getFullYear().toString());
```

Comment:
This only works, because we are checking for the year, if check for seconds or milliseconds this will fail. Testing dates this way, leads to instable tests. We should discuss a way around this.